en/SettingsView/csv.md

```diff
@@ -0,0 +1,12 @@
+# CSV Logging
+
+> **Note** This is a daily build feature (not present in QGC v3.5)
+
+![Csv checkbox](../../assets/settings/general/csv.jpg)
+
+By checking this option, whenever a vehicle is connected and armed (unless "Save logs even if vehicle was not armed" is checked) a CSV (Comma-separated values) file will be created alongside the usual .tlog telemetry file.
+
+This CSV file contains the data from the vehicle Facts, which includes most of the relevant flight data quick analysis such as position, attitude, battery status, and others.
```

Comment:
```suggestion
This CSV file contains the most relevant vehicle telemetry data available for quick analysis such as GPS position, attitude, battery status, and others.
```

Reply:
@Williangalvani I don't have permission to push to this repo for some reason. But I think you should accept this.